src/Parquet.chpl

```diff
@@ -0,0 +1,109 @@
+module Parquet {
+  use SysCTypes, CPtr, IO;
+  use ServerErrors, ServerConfig;
+  if hasParquetSupport {
+    require "ArrowFunctions.h";
+    require "ArrowFunctions.o";
+  }
+
+  private config const ROWGROUPS = 512*1024*1024 / numBytes(int); // 512 mb of int64
+
+  extern var ARROWINT64: c_int;
+  extern var ARROWINT32: c_int;
+  extern var ARROWUNDEFINED: c_int;
+
+  enum ArrowTypes { int64, int32, notimplemented };
+  
+  proc getVersionInfo() {
+    extern proc c_getVersionInfo(): c_string;
+    extern proc strlen(str): c_int;
+    extern proc c_free_string(ptr);
+    var cVersionString = c_getVersionInfo();
+    defer {
+      c_free_string(cVersionString: c_void_ptr);
+    }
+    var ret = try! createStringWithNewBuffer(cVersionString,
+                                             strlen(cVersionString));
```

Comment:
If this fails it takes out the server process.  Should we throw an error here instead?